src/main/markdown/doc/latest/tutorial/manageevents.md

```diff
@@ -129,36 +129,37 @@ For the enter key, you however have to use either KeyDownHandler or KeyUpHandler
 1.  Hook up the keydown event handler for the input box, newSymbolTextBox.
     *  In the onModuleLoad method, cut and paste the code commented "Listen for keyboard events in the input box." that is highlighted below.
 
-```
-// Listen for mouse events on the Add button.
-    addStockButton.addClickHandler(new ClickHandler() {
-      public void onClick(ClickEvent event) {
-        addStock();
-      }
-    });
-
-        // Listen for keyboard events in the input box.
-    newSymbolTextBox.addKeyDownHandler(new KeyDownHandler() {
-      public void onKeyDown(KeyDownEvent event) {
-        if (event.getNativeKeyCode() == KeyCodes.KEY_ENTER) {
-          addStock();
-        }
-      }
-    });
-
-      }
-
-      /**
-       * Add stock to FlexTable. Executed when the user clicks the addStockButton or
-       * presses enter in the newSymbolTextBox.
-   */
-  private void addStock() {
-    // TODO Auto-generated method stub
-
-      }
-
-    }
-```
+   ```
```

Comment:
Apologies. I did not see a consistent indentation in the file (4 for ordered lists, 3 for unordered, so nesting gets complicated). The takeaway is that the "Handling Keyboard Events" section currently renders two items numbered as "1" and renders an unordered list item as code.

That dangling curly brace (presumably for closing the class) complicates matters somewhat, but I will try to revise.